src/transceivercontroller/VideoOnlyTransceiverController.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import BrowserBehavior from '../browserbehavior/BrowserBehavior';
+import Logger from '../logger/Logger';
+import DefaultTransceiverController from './DefaultTransceiverController';
+
+export default class VideoOnlyTransceiverController extends DefaultTransceiverController {
+  constructor(logger: Logger, browserBehavior: BrowserBehavior) {
+    super(logger, browserBehavior);
+  }
+
+  setupLocalTransceivers(): void {
```

Comment:
`setUp`: https://grammarist.com/spelling/set-up-vs-setup/

Reply:
this is an interface method. We probably don't want to change it until v3.

Reply:
Ah, I missed that it was inherited. Thanks!